packages/phpstan-rules/tests/Rules/Explicit/NoVoidAssignRule/Fixture/SomeVoidAssign.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\Explicit\NoVoidAssignRule\Fixture;
+
+final class SomeVoidAssign
+{
+    public function run()
+    {
+        $value = $this->getNothing();
+    }
+
+    public function getNothing(): void
```

Comment:
Should/Will the rule also work for `@return void` phpdoc-annotated methods?

Reply:
Depends on PHPStan, I'll add more fixtures